command/operator_debug.go

```diff
@@ -399,6 +415,31 @@ func (c *OperatorDebugCommand) Run(args []string) int {
 		c.Ui.Error(fmt.Sprintf("Error parsing arguments: %q", err))
 		return 1
 	}
+	// Parse logLookback and logFileExport to set export string with whichever is set
+	if logLookback != "" && logFileExport == "" {
+		_, err := time.ParseDuration(logLookback)
+		if err != nil {
+			c.Ui.Error(fmt.Sprintf("Error parsing log-lookback value: %s: %s", logLookback, err.Error()))
+			return 1
+		}
+		export = logLookback
+	}
+
+	if logFileExport != "" && logLookback == "" {
+		t, err := strconv.ParseBool(logFileExport)
+		if err != nil {
+			c.Ui.Error(fmt.Sprintf("Error parsing log-file-export value: %s: %s", logFileExport, err.Error()))
+			return 1
+		}
+		if t {
+			export = logFileExport
+		}
+	}
+
+	if logFileExport != "" && logLookback != "" {
+		c.Ui.Error("Error parsing inputs, -log-file-export and -log-lookback cannot be used together.")
+		return 1
+	}
```

Comment:
This args parsing approach seems convoluted and we end up having to reparse the arguments in the `startMonitorExport` method because we're trying to reuse the `export` variable for either code path. I think we can just send the parameters through and let the downstream functions handle what to do with them.

Reply:
Thanks @tgross ! That makes sense. You're right that I was hesitant to add both variables to the `collect` function but it's not like`collect` is being used outside this file so I can see how it's not worth adding 30 repetitive lines to the top of the file to minimize the change to that function's signature.